src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -123,7 +123,8 @@
       "GET_MANAGED_USER": "user/v1/managed",
       "USER_EXISTS_GET_USER_BY_KEY": "isUserExists/user/v1/get",
       "USER_DECLARATION": "user/v1/declarations",
-      "GET_SESSION": "get/user/sessionId"
+      "GET_SESSION": "get/user/sessionId",
+      "READ": "user/v1/read"
```

Comment:
@snehalsabade No need to add "READ" in the upgrade changed end point with "GET_PROFILE"

Reply:
resolved